packages/core-ethereum/crates/core-ethereum-db/src/db.rs

```diff
@@ -853,6 +856,28 @@ pub mod wasm {
         }
     }
 
+    macro_rules! check_lock_read {
+        { $($rest:tt)* } => {
+            let r = {
+                utils_misc::console_log!("{} >>> READ ", stdext::function_name!());
+                $($rest)*
+            };
+            utils_misc::console_log!("{} <<< READ ", stdext::function_name!());
+            r
+        };
+    }
+
+    macro_rules! check_lock_write {
+        { $($rest:tt)* } => {
+            let r = {
+                utils_misc::console_log!("{} >>> WRITE ", stdext::function_name!());
+                $($rest)*
+            };
+            utils_misc::console_log!("{} <<< WRITE ", stdext::function_name!());
+            r
+        };
+    }
+
```

Comment:
Are these still needed?

Reply:
Going to comment these out, so they could be used eventually if needed